Feature :Added Badge Component

kiranpratyush opened the pull request:
### updated index.html according to the documentation site(though some changes are pending for header ignore it for now)
### updated the style.css
### Below is the live working link please review it and give help full feedback

## Components Defined - 
- Badge


## Live Link -   [Testing](https://testing2311.netlify.app/badge/)

kiranpratyush committed:
Feat:Added Badge Component

jayeshjain98 commented on Badge/index.html:
```diff
@@ -0,0 +1,31 @@
+<div>
+    <img src="https://robohash.org/pratyush.png?set=set2" alt="robot image" class="avatar"> <span class="badge" style ="transform: translateX(-30px);";">2</span>
+<div/>
+        <p>Below is a demonstration how you can use badge with icontransform property</p>
+    <i class="bi bi-bag-fill" style="font-size: 70px;"><span class="badge" style ="transform: translate(-20px,10px);";">2</span></i>
```
i think you should remove inline style because it will not be available to the user of your CL.

kiranpratyush replied:
Thank you for the feedback. I'll make the changes.

kiranpratyush committed:
Feat:Updated index.html and style.css

Updated index.html according to the documentation website and refactored style.css

preethasri commented:
UI looks clean  Kiran 👍
